### ISSUE TITLE
The script contains outdated/non-existing packages and is missing recently released free packages

### ISSUE THREAD

**Issue by louisa-uno** [5x/easy-steam-free-packages#12]
### This issue is based on [this discussion](https://github.com/5x/easy-steam-free-packages/discussions/11):

<div type='discussions-op-text'>

<sup>Originally posted by **dafunks** January 18, 2022</sup>
I always find myself checking back here to see if any updates have happened with new games or software. No updates for a few years now. Any plans to add anything? 

Thanks for the work so far as its given a lot to my account.</div>

**Comment by JoshuaVandaele** [comment]
I don't think this script will get updated, but you can use this tool made available by SteamDB in the meantime: https://steamdb.info/freepackages/
